.github/ISSUE_TEMPLATE/-maintainer--new-feature.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "(Maintainer) New Feature"
+about: New feature description, filed by CRM maintainers
+title: ''
+labels: enhancement, source::maintainer
+assignees: ''
+
+---
+
+**What is the idea?**
+Describe what the feature is and the desired state.
+
+**Why is this needed?**
+Who would benefit from this feature? Why would this add value to them? What problem does this solve?
+
+**Alternatives that have been considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
```

Comment:
Seems a little redundant with the first question.

That being said. Maybe we should have an acceptance criteria section? I probably won't fill it out a lot of the time, but it might be nice to have when we want to be clear about what the expectation is around testing/"what counts as complete"?